cms/djangoapps/export_course_metadata/storage.py

```diff
@@ -0,0 +1,20 @@
+"""
+Storage backend for course metadata export.
+"""
+
+
+from django.conf import settings
+from django.core.files.storage import get_storage_class
+from storages.backends.s3boto import S3BotoStorage
+
+
+class CourseMetadataExportS3Storage(S3BotoStorage):  # pylint: disable=abstract-method
```

Comment:
i haven't actually tested this, though i copied the pattern from the course import export that also uploads to S3.  
devops recommended trying the code out with a waffle flag rather than trying to connect to S3 from devstack